6tisch-roll-enrollment-priority.mkd

```diff
@@ -82,92 +82,111 @@ There are a variety of local reasons by which a 6LR might not want to provide th
-The contents of this option are not considered to be a change, and do not reset the trickle timer.
+Updates to the option propagate through the network according to the trickle algorithm.
+The contents of the option are generated at the DODAG Root and do not change at any hop.
+If the contents represent an update that is considered important (e.g., quickly disabling any enrollments), the option can trigger trickle timer resets at the nodes to speed up its propagation.
 
 # Terminology          {#Terminology}
 
 {::boilerplate bcp14}
 
 The term (1)"Join" has been used in documents like {{!RFC9031}} to denote the activity of a new node authenticating itself to the network in order to obtain authorization to become a member of the network.
 
-In the context of the {{!RFC6550}} RPL protocol, the term (2)"Join" has an alternate meaning: that of a node (already authenticating to the network, and already authorized to be a member of the network), deciding which part of the RPL DODAG to attach to.
+In the context of the {{!RFC6550}} RPL protocol, the term (2)"Join" has an alternative meaning: that of a node (already authenticating to the network, and already authorized to be a member of the network), deciding which part of the RPL DODAG to attach to.
 This term "Join" has to do with preferred parent selection processes.
 
 In order to avoid the ambiguity of this term, this document refers to the process (1)"Join" as enrollment, leaving the term "Join" to mean (2)"Join".
 The term "onboarding" (or IoT Onboarding) is increasingly used to describe what was called enrollment in other documents.
-However, the term *Join Proxy* is retained with it's meaning from {{!RFC9031}}.
+However, the term *Join Proxy* is retained with its meaning from {{!RFC9031}}.
 
 
 # Protocol Definition
 
-With this specification, the following option is defined for transmission in the DIO issued by the DODAG root and it MUST be propagated down the DODAG.
-
-A 6LR which would otherwise be willing to act as a *Join Proxy*, will examine the minimum priority field, and to that number, add any additional local consideration (such as upstream congestion, number of NCE slots available, etc.)
+This document uses the extensions mechanism designed into {{!RFC6550}}.
+No mechanism is needed to enable it.
 
-The Enrollment Priority can only be increased by each 6LR in value, to the maximum value of 0x7f.
+## Option Format
 
-The resulting priority, if less than 0x7f should enable the *Join Proxy* function.
+The following option is defined for transmission in DIOs issued by the DODAG root to be propagated within the DODAG.
 
         0                   1                   2                   3
         0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1
        +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
-       |   Type        |Opt Length = 3 |  exp  |     DODAG_Size        |
+       | Type = TBD01  |Opt Length = 4 |Version Number |T| min priority|
```

Comment:
Yes, precisely.  I assumed that it is a network operator's decision whether the bit should be set. Nevertheless, it may be worthwhile to give some examples for implementers when the bit can be set, such as a change from maximal value to nonmaximal or vice versa.